src/Infrastructure/Form/Regulation/GeneralInfoFormType.php

```diff
@@ -63,6 +64,19 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'choice_label' => 'name',
                 ],
             )
+            ->add(
+                'category',
+                ChoiceType::class,
+                options: $this->getCategoryOptions(),
+            )
+            ->add(
+                'otherCategoryText',
+                TextType::class,
+                options: [
+                    'required' => false,
+                    'label' => 'regulation.general_info.otherCategoryText',
```

Comment:
(Pinaillage) Respecter le snake_case ?

```suggestion
                    'label' => 'regulation.general_info.other_category_text',
```